src/test/groovy/edu/ie3/datamodel/io/processor/input/AssetInputProcessorTest.groovy

```diff
@@ -0,0 +1,286 @@
+package edu.ie3.datamodel.io.processor.input
+
+import edu.ie3.datamodel.models.StandardUnits
+import edu.ie3.datamodel.models.input.NodeInput
+import edu.ie3.datamodel.models.input.connector.LineInput
+import edu.ie3.datamodel.models.input.connector.SwitchInput
+import edu.ie3.datamodel.models.input.connector.Transformer2WInput
+import edu.ie3.datamodel.models.input.connector.Transformer3WInput
+import edu.ie3.datamodel.models.input.system.BmInput
+import edu.ie3.datamodel.models.input.system.ChpInput
+import edu.ie3.datamodel.models.input.system.EvInput
+import edu.ie3.datamodel.models.input.system.FixedFeedInInput
+import edu.ie3.datamodel.models.input.system.HpInput
+import edu.ie3.datamodel.models.input.system.LoadInput
+import edu.ie3.datamodel.models.input.system.PvInput
+import edu.ie3.datamodel.models.input.system.StorageInput
+import edu.ie3.datamodel.models.input.system.WecInput
+import edu.ie3.test.common.GridTestData
+import edu.ie3.test.common.SystemParticipantTestData
+import edu.ie3.util.TimeTools
+import spock.lang.Specification
+
+import java.time.ZoneId
+import java.time.ZonedDateTime
+
+/**
+ * //ToDo: Class Description
+ *
+ * @version 0.1* @since 24.03.20
+ */
+class AssetInputProcessorTest extends Specification {
+
+
+    def "A ResultEntityProcessor should de-serialize a provided NodeInput correctly"() {
+        given:
+        TimeTools.initialize(ZoneId.of("UTC"), Locale.GERMANY, "yyyy-MM-dd HH:mm:ss")
+        def sysPartResProcessor = new AssetInputProcessor(NodeInput)
+        def validResult = GridTestData.nodeA
+
+        Map expectedResults = [
+                "uuid"         : "5dc88077-aeb6-4711-9142-db57292640b1",
+                "geoPosition"  : "{\"type\":\"Point\",\"coordinates\":[7.411111,51.492528],\"crs\":{\"type\":\"name\",\"properties\":{\"name\":\"EPSG:4326\"}}}",
+                "id"           : "node_a",
+                "operatesUntil": "2020-03-25 15:11:31",
+                "operatesFrom" : "2020-03-24 15:11:31",
+                "operator"     : "8f9682df-0744-4b58-a122-f0dc730f6510",
+                "slack"        : "true",
+                "subnet"       : "1",
+                "vTarget"      : "1.0",
+                "voltlvl"      : "Höchstspannung (380 kV)",
```

Comment:
Just for homogeneity: At other places I expect the voltage level to be of the form `"{Höchstspannung, 380 kV}"`. Additionally: Shouldn't this be de-serialized to
```
"voltlvl"      : "Höchstspannung"
"vRated"    : "380 kV"
```
?

Reply:
No. ;-)

we should have a short talk about this then.

tl;dr-version:

1. voltLvl is the `id` String instance you placed in `GermanVoltageLevelUtils`. After our short discussion yesterday this is (from a serialized perspective) equal to one column in a resulting .csv-file or database field named `voltLvL`. We agreed yesterday to keep this structure to not introduce new columsn/fields. If you think it makes sense to change this again let me know. As discussed several times previously, I'm not a big fan of putting json strings inside column/row based data structures as this can become very hard to read quickly
2. Your second commend is correct, IF we want to introduce two new fields in our datastructs. If you think this makes sense, we should discuss this. 
3. Where du you expect `{Höchstspannung, 380 kV}` in other places? As this is neither standard json encoding nor something else I would highly recommend changing this to a clearer (maybe more suitable) structure.